src/bio/bio_xstream.c

```diff
@@ -367,8 +371,27 @@ bio_nvme_init(const char *nvme_conf, int shm_id, int mem_size,
 		goto free_mutex;
 	}
 
+	D_ASSERT(tgt_nr > 0);
+	D_ASSERT(mem_size > 0);
```

Comment:
Hit NLT error:
`
 ERROR: daos_engine:0 06/08-23:13:51.51 5607cdad6a98 DAOS[33818/0/0] bio  EMRG src/bio/bio_xstream.c:375 bio_nvme_init() Assertion 'mem_size > 0' failed
  ERROR: daos_engine:0 daos_engine: src/bio/bio_xstream.c:375: int bio_nvme_init(const char *, int, int, int, int, struct sys_db *): Assertion `mem_size > 0' failed.
  ERROR: daos_engine:0 *** Process 33818 received signal 6 ***

Seems the 'mem_size' isn't properly passed in? Or we don't configure hugepages in some tests?